monkey/infection_monkey/monkey.py

```diff
@@ -189,10 +193,20 @@ def _get_local_network_interfaces():
 
         return local_network_interfaces
 
-    @staticmethod
-    def _build_puppet() -> IPuppet:
+    def _build_puppet(self) -> IPuppet:
         puppet = Puppet()
 
+        puppet.load_plugin(
+            "MimikatzCollector",
+            MimikatzCredentialCollector(),
+            PluginType.CREDENTIAL_COLLECTOR,
+        )
```

Comment:
Or maybe we should check OS here, maybe we shouldn't even load plugins that can't run on the current OS (now is not the time to refactor probably)